bundle/metadata/dependencies.yaml

```diff
@@ -0,0 +1,5 @@
+dependencies:
+- type: olm.package
+  value:
+    packageName: ramen
+    version: "4.11.0"
```

Comment:
```
  packageName: ramen-hub-operator
  version: "0.0.1"
```
Test builds are published with these details. It will change for stable builds.
We need to make these values replaceable at build time and use these values as defaults.